packages/atomic/src/utils/result-utils.ts

```diff
@@ -42,3 +43,36 @@ export function bindLogDocumentOpenOnResult(
     });
   };
 }
+
+export function buildStringTemplateFromResult(
+  template: string,
+  result: Result,
+  bindings: Bindings
+) {
+  return template.replace(/\${(.*?)}/g, (value: string) => {
+    const key = value.substring(2, value.length - 1);
+    let newValue = readFromObject(result, key);
```

Comment:
we use lodash in this project, right? how about `_.get(result, key)` which then translates to, say, `get(result, 'raw.title')` instead of this new func?

Reply:
We use lodash only as a dev dependencies for tests IIRC